AutomatedTesting/Levels/Prefab/Base/Base.prefab

```diff
@@ -28,14 +28,15 @@
                 "$type": "EditorDisabledCompositionComponent",
                 "Id": 16239496886950819870
             },
+            "Component_[3956821497370414653]": {
+                "$type": "LocalViewBookmarkComponent",
+                "Id": 3956821497370414653,
+                "LocalBookmarkFileName": "Base_852100585203.setreg"
```

Comment:
fyi, if tests run in parallel, writing to the same file may become an issue. Although, that should only happen for bookmark tests or tests that write/read from that file.